national-insurance.conf

```diff
@@ -6811,6 +6811,65 @@ category-names {
     }
 }
 
+2021 {
+  limits {
+    # we can explicitly give values for weeks, months and 4weeks here
+    # but if we omit them it will get them from the year divided by 52, 12 and 13 respectively
+    LEL.year = £6240
+    ST.year = £8840
+    PT.year = £9568
+    UEL.year = £50270
+  }
+
+  class-one {
+  
+    # the intervals for the bands are inferred by the names.
+    # in this case 'up to X' will cause the configuration to look for a value for 'X'
+    # under limits above
+    "Up to LEL" {
+      # defining a 0% band will cause it to appear in the tables
+      employee.AHMBCJZ = 0% 
+      employer.ABCJMZ = 0%
+    }
+    "LEL to ST".employee.AHMBCJZ = 0%
+    "LEL to PT".employer.ABCJ = 13.8%
+    "PT to UEL" {
+      employee {
+        AHM = 12%
+	B = 5.85%
+	JZ = 2%
```

Comment:
`Up to LEL`, `LEL to PT` and `PT to UEL` now need `hide-on-summary: false` on them now to get them displaying properly on the C1 print screen

Reply:
done.